skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -161,107 +156,102 @@ private function getCuratedContentButton() {
 	}
 
 	/**
-	 * Render default page header (with edit dropdown, history dropdown, ...)
+	 * Get the page title
+	 * Used by executeIndex and executeCorporate
 	 *
-	 * @param: array $params
-	 *    key: showSearchBox (default: false)
+	 * @return string $title
 	 */
-	public function executeIndex( $params ) {
-		global $wgTitle, $wgArticle, $wgOut, $wgUser, $wgContLang, $wgSupressPageTitle, $wgSupressPageSubtitle,
-			$wgSuppressNamespacePrefix, $wgEnableWallExt;
-
-		wfProfileIn( __METHOD__ );
-
-		$this->isUserLoggedIn = $wgUser->isLoggedIn();
-
-		// check for video add button permissions
-		$this->showAddVideoBtn = $wgUser->isAllowed( 'videoupload' );
-
-		// page namespace
-		$ns = $wgTitle->getNamespace();
-
-		$this->curatedContentToolButton = $this->getCuratedContentButton();
-
-		/** start of wikia changes @author nAndy */
-		$this->isWallEnabled = ( !empty( $wgEnableWallExt ) && $ns == NS_USER_WALL );
-		/** end of wikia changes */
-
-		// currently used skin
-		$skin = RequestContext::getMain()->getSkin();
-
-		// action button (edit / view soruce) and dropdown for it
-		$this->prepareActionButton();
-
-		// dropdown actions
-		$this->dropdown = $this->getDropdownActions();
-
-		/** start of wikia changes @author nAndy */
-		$response = $this->getResponse();
-		if ( $response instanceof WikiaResponse ) {
```

Comment:
It was only used for hooks which have been altered to only allow what they should, e.g. passing only the necessary variables to them. See `PageHeaderIndexAfterActionButtonPrepared` and `PageHeaderIndexExtraButtons`.